tools/modules/software/module_samba.sh

```
@@ -11,117 +11,69 @@ module_options+=(
+		return 1
+	else
+		echo "$title service is in an unknown state."
+		return 1
+	fi
+}
+
+function module_samba() {
+	local title="samba"
+	local feature="module_samba"
+	local condition
+	condition=$(command -v smbd)
```

Shellcheck (0.10.0)

[warning] condition appears unused. Verify use (or export if used externally).

(SC2034)